test/system/250-systemd.bats

```diff
@@ -292,4 +292,62 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
     run_podman network rm -f $netname
 }
 
+# NOTE - running the test locally requires the `podman-play-kube@.service` file
+# **and** the `podman` binary to be installed.
+@test "podman-play-kube@.service template" {
+    skip_if_remote "systemd units do not work with remote clients"
+
+    # Create the YAMl file
+yaml="
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: test
+  name: test_pod
+spec:
+  containers:
+  - command:
+    - top
+    image: $IMAGE
+    name: test
+    resources: {}
+"
+    yaml_source="$PODMAN_TMPDIR/test.yaml"
+    echo "$yaml" > $yaml_source
+
+    # Dispatch the YAML file
+    systemctl start podman-play-kube@$(systemd-escape $yaml_source).service
+    systemctl is-active podman-play-kube@$(systemd-escape $yaml_source).service
+
```

Comment:
I count four instances of `podman-play-etc-escape-etc`, which makes it a good candidate for refactoring into a short variable name